bindings/java/src/test/java/org/openucx/jucx/UcpEndpointTest.java

```diff
@@ -447,4 +442,57 @@ public void onSuccess(UcpRequest request) {
             serverToClient);
         closeResources();
     }
+
+    @Test
+    public void testEpErrorHandler() {
+        // Crerate 2 contexts + 2 workers
+        UcpParams params = new UcpParams().requestTagFeature();
+        UcpWorkerParams workerParams = new UcpWorkerParams();
+        UcpContext context1 = new UcpContext(params);
+        UcpContext context2 = new UcpContext(params);
+        UcpWorker worker1 = context1.newWorker(workerParams);
+        UcpWorker worker2 = context2.newWorker(workerParams);
+
+        ByteBuffer src = ByteBuffer.allocateDirect(UcpMemoryTest.MEM_SIZE);
+        ByteBuffer dst = ByteBuffer.allocateDirect(UcpMemoryTest.MEM_SIZE);
+        src.asCharBuffer().put(UcpMemoryTest.RANDOM_TEXT);
+
+        AtomicBoolean errorHandlerCalled = new AtomicBoolean(false);
+        UcpEndpointParams epParams = new UcpEndpointParams()
+            .setPeerErrorHadnlingMode()
+            .setErrorHandler((ep, status, errorMsg) -> errorHandlerCalled.set(true))
```

Comment:
Is it useful to set peer error handling mode and not set a callback? e.g. should `.setErrorHandler` also set the peer error handling mode to `UCP_ERR_HANDLING_MODE_PEER` and therefore avoid the second call to `setPeerErrorHadnlingMode`?